debezium/dbz#1903 Document metrics quantiles (#7426)
debezium/debezium: documentation/modules/ROOT/partials/modules/all-connectors/ref-connector-monitoring-streaming-metrics.adoc

|The number of milliseconds between the last change event's timestamp and the connector processing it.
The values will incorporate any differences between the clocks on the machines where the database server and the connector are running.

|[[connectors-strm-metric-millisecondsbehindsourceminvalue_{context}]]<<connectors-strm-metric-millisecondsbehindsourceminvalue_{context}, `MilliSecondsBehindSourceMinValue`>>
|`long`

> |The minimum lag in milliseconds behind the source that has been observed during the connector's runtime.

Reviewer: ```suggestion
|The minimum lag in milliseconds behind the source that is observed during the connector's runtime.
```

|[[connectors-strm-metric-millisecondsbehindsourcemaxvalue_{context}]]<<connectors-strm-metric-millisecondsbehindsourcemaxvalue_{context}, `MilliSecondsBehindSourceMaxValue`>>
|`long`

> |The maximum lag in milliseconds behind the source that has been observed during the connector's runtime.

Reviewer: ```suggestion
|The maximum lag in milliseconds behind the source that is observed during the connector's runtime.
```

|[[connectors-strm-metric-millisecondsbehindsourceaveragevalue_{context}]]<<connectors-strm-metric-millisecondsbehindsourceaveragevalue_{context}, `MilliSecondsBehindSourceAverageValue`>>
|`double`
|The average lag in milliseconds behind the source calculated across all observations during the connector's runtime.

|[[connectors-strm-metric-millisecondsbehindsourcep50_{context}]]<<connectors-strm-metric-millisecondsbehindsourcep50_{context}, `MilliSecondsBehindSourceP50`>>
|`double`
|The 50th percentile (median) of the lag in milliseconds behind the source.

> This metric provides a more robust measure of typical lag than the average, as it is less affected by outliers.

Reviewer: ```suggestion
This metric provides a more robust measure of typical lag than the average, as it is less affected by outlier values.
```
